src/util/network.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_UTIL_NETWORK_H
+#define BITCOIN_UTIL_NETWORK_H
+
+#include <string_view>
+
+enum class SocketAddrError {
```

Comment:
The rest of the codebase is using just "address" or "network address" (to distinguish from a bitcoin address), or host. Introducing a new name for the same thing could be confusing.

Reply:
thanks